pkg/plugins/message/message.go

```diff
@@ -108,14 +107,3 @@ func (m *Message) Nack() bool {
 func (m *Message) Nacked() <-chan struct{} {
 	return m.nack
 }
-
-// FromByte is returning an enfolded message from the queue.
-func FromByte(b []byte) (*Message, error) {
-	msg := new(Message)
-
-	if err := json.Unmarshal(b, msg); err != nil {
-		return nil, err
-	}
-
-	return msg, nil
-}
```

Comment:
think we should be able to remove this structure 🤔

Reply:
Yeah, I think we may use it for internal representation. Because the ack information maybe handy for features. Wdyt?